### ISSUE TITLE
Facebook IFTTT recipe fix

### ISSUE THREAD

**Issue by jsatk** [ttscoff/Slogger#137]
When following the instructions for the Facebook IFTTT recipe I kept getting the error: `/Users/Jesse/Library/Scripts/Slogger/plugins/facebookifttt.rb:83:in`do_log': undefined method `each' for #<String:0x007fb28adec760> (NoMethodError)`.

Turns out that Ruby 1.8's String.each method is depreciated and now if you're using Ruby 1.9 it uses String.each_line ([reference](https://groups.google.com/forum/?fromgroups=#!topic/rubyonrails-talk/Ed7CN1RRexs)).

Not sure if this is something that _should_ be updated or not. I'd imagine plenty of folk are still on Ruby 1.8 and if they pull the latest **Slogger** and it breaks on them they'd be unhappy. But, ultimately that's your call.

Long story short: Changing line 83 of `facebookifttt.rb` from `content.each do |line|
` to `content.each_line do |line|` fixes the error.
